frontend/class-twitter.php

```diff
@@ -219,6 +243,9 @@ protected function title() {
 		elseif ( WPSEO_Frontend::get_instance()->is_posts_page() ) {
 			$title = $this->single_title( get_option( 'page_for_posts' ) );
 		}
```

Comment:
Same as comment above: I think this would be more readable in a switch statement